### ISSUE TITLE
incorrect-spdx skips single-segment +spdx values like 'MIT'

### ISSUE THREAD

**Issue by VasilevNStas** [objectionary/lints#1382]
## Problem

`incorrect-spdx` does not check single-segment `+spdx` values at all — e.g. `+spdx MIT` (with no space-separated `SPDX-...:` header) is silently accepted.

## Root cause

`src/main/resources/org/eolang/lints/metas/incorrect-spdx.xsl:13`:

```xml
<xsl:for-each select="/object/metas/meta[head='spdx' and count(part) > 1]">
```

The `count(part) > 1` guard skips metas whose tail has a single whitespace-separated part, so `+spdx MIT` never reaches the validation logic.

## Minimal example

```eo
+package x
+architect yegor@256.com
+version 0.0.1
+home https://github.com/objectionary
+spdx MIT
+spdx SPDX-FileCopyrightText: Copyright (c) 2026 Objectionary.com
+spdx SPDX-License-Identifier: MIT

[] > main
```

`+spdx MIT` is not flagged, even though a valid SPDX meta must start with `SPDX-...:`.

## Expected behavior

Single-part `+spdx` values (e.g. `MIT`, `Apache-2.0`) should be flagged as incorrect, or the guard should be reworked so the validation covers them.

## Related

- `metas/mandatory-spdx.xsl` — sibling rule.
- Existing packs under `packs/single/incorrect-spdx/`.

**Comment by 0crat** [comment]
@vasilevnstas Thanks for reporting a new bug! You've earned +12 points. By reporting bugs, you help our project improve its quality. If you find anything else in the repository that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) as good as you might expect, [do not hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) it. Your running score is [+806](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).
